.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+jobs:
+  build:
```

Comment:
this is not `build`, this is `install` or something :thinking: `build` would be added assuming we end up bulding the app after tests are done. Not major, but consider this. In the meantime though, feel free to merge.